### PR TITLE
OpenAI Integration

### PR DESCRIPTION
**Pull request by wstinkens**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
OpenAI integration which interacts with OpenAI Completion API. 

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [X] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

File: Packs/OpenAI/Integrations/OpenAI/OpenAI.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+''' COMMAND FUNCTIONS '''
+
+
+def test_module_command(client):
+    """
+    Tests OpenAPI connectivity
+    """
+    result = client.completions(prompt="Can I connect to the OpenAI api?")
+    if result:
+        return 'ok'
```
Comment (ostolero) [inline Packs/OpenAI/Integrations/OpenAI/OpenAI.py:73]: if no result is returned, return a relevant error.
for example-
    if result:
        return 'ok'
    else:
        return 'Error with {sonme indicative message}'

Comment (wstinkens) [reply Packs/OpenAI/Integrations/OpenAI/OpenAI.py:73]: Requested change has been implemented

**Review by ostolero — APPROVED** [review]
Hi @wstinkens the code looks great! really well done, and thank you for the quick response.
Let's schedule our demo, I'll reach out through slack.
Thanks!

**Comment by ostolero** [comment]
Demo went great! merging :)

**Pull request merged**